src/components/AcceleratorDeliverableView/DeliverableCard.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { ViewProps } from '../DeliverableView/types';
+import DocumentDeliverableCard from './DocumentDeliverableCard';
+import QuestionsDeliverableCard from './QuestionsDeliverableCard';
+import SpeciesDeliverableCard from './SpeciesDeliverableCard';
+
+const DeliverableCard = ({ deliverable }: ViewProps) => {
+  if (!deliverable) {
+    return undefined;
+  }
+
+  return (
+    <>
+      {deliverable.type === 'Document' ? (
+        <DocumentDeliverableCard deliverable={deliverable} />
+      ) : deliverable.type === 'Questions' ? (
+        <QuestionsDeliverableCard deliverable={deliverable} />
+      ) : (
+        <SpeciesDeliverableCard deliverable={deliverable} />
```

Comment:
Maybe a `switch` or `if...elseif` would be a bit more readable here. Also if the type is unknown/unexpected, we should not render the species deliverable card.